src/survey/constants.py

```diff
@@ -6,3 +6,45 @@
 STRONGLY_GOOD = 5
 
 RATINGS = [STRONGLY_BAD, SLIGHTLY_BAD, NEUTRAL, SLIGHTLY_GOOD, STRONGLY_GOOD]
+
+BAD_FIT = ("BF", "Not a good fit")
+NO_RESPONSE = ("NR", "They didn't respond")
+NOT_INTERESTED = ("NI", "Not interested")
+TOO_BUSY = ("TB", "Too busy")
+OTHER = ("OT", "Other")
+
+DID_NOT_MEET = [BAD_FIT, NO_RESPONSE, NOT_INTERESTED, TOO_BUSY, OTHER]
+DID_NOT_MEET_SHORT = list(map(lambda x: x[0], DID_NOT_MEET))
+DID_NOT_MEET_LONG = list(map(lambda x: x[1], DID_NOT_MEET))
+
+
+def short_to_long(short_name):
+    """Convert a short name to a long name"""
+    if short_name == BAD_FIT[0]:
+        return BAD_FIT[1]
+    elif short_name == NO_RESPONSE[0]:
+        return NO_RESPONSE[1]
+    elif short_name == NOT_INTERESTED[0]:
+        return NOT_INTERESTED[1]
+    elif short_name == TOO_BUSY[0]:
+        return TOO_BUSY[1]
+    elif short_name == OTHER[0]:
+        return OTHER[1]
+    else:
+        return None
+
+
+def long_to_short(long_name):
+    """Convert a long name to a short name"""
+    if long_name == BAD_FIT[1]:
+        return BAD_FIT[0]
+    elif long_name == NO_RESPONSE[1]:
+        return NO_RESPONSE[0]
+    elif long_name == NOT_INTERESTED[1]:
+        return NOT_INTERESTED[0]
+    elif long_name == TOO_BUSY[1]:
+        return TOO_BUSY[0]
+    elif long_name == OTHER[1]:
+        return OTHER[0]
+    else:
+        return None
```

Comment:
what is the point of converting from long to short name
why can't it just be a dictionary mapping too, then it can just return the `value` given the `key`

Reply:
These functions could've been simplified with a loop checking DID_NOT_MEET rather than the tuple values. I agree with @alanna-zhou  about the dictionary, just make one and make the other by reversing the keys and values and choose the appropriate dictionary when converting.

Reply:
good call